### PR TITLE
fix: update schematics to match the last updates with traxion best practices

### PR DESCRIPTION
**Pull request by floross**
- chore: remove create-traxion-workspace from the repository
- WIP: clean the schematics to respect our new organisation
- wip: update schematics

### CODE REVIEW

File: libs/schematics/src/generators/traxion-workspace/helpers/create-admin-application.helper.ts [hunk 1]
```diff
@@ -7,10 +7,10 @@ export async function createAdminApplication(
   tree: Tree,
   options: NormalizedOptions,
 ) {
-  const { adminName, generatedImportPath } = options;
-  await adminGenerator(tree, {
-    name: adminName,
-    reactAdminImportPath: `${generatedImportPath}react-admin`,
-    rextClientImportPath: `${generatedImportPath}rext-client`,
-  });
+  // const { adminName, generatedImportPath } = options;
```
Comment (EdouardDem) [inline libs/schematics/src/generators/traxion-workspace/helpers/create-admin-application.helper.ts:15]: Is it intended that this code remains commented on?

File: libs/schematics/src/generators/traxion-workspace/helpers/create-angular-application.helper.ts [hunk 2]
```diff
@@ -7,11 +7,11 @@ export async function createAngularApplication(
...
-    generatedDir,
-    apiName,
-  });
+  // const { pwaName, generatedDir, apiName, directory } = options;
+  // await pwaAppGenerator(tree, {
+  //   name: pwaName,
+  //   directory,
+  //   generatedDir,
+  //   apiName,
+  // });
```
Comment (EdouardDem) [inline libs/schematics/src/generators/traxion-workspace/helpers/create-angular-application.helper.ts:20]: Is it intended that this code remains commented on?

File: libs/schematics/src/generators/traxion-workspace/helpers/create-nestjs-application.helper.ts [hunk 3]
```diff
@@ -7,10 +7,10 @@ export async function createNestjsApplication(
...
-    name: apiName,
-    directory,
-    generatedDir,
-  });
+  // const { directory, apiName, generatedDir } = options;
+  // await apiAppGenerator(tree, {
+  //   name: apiName,
+  //   directory,
+  //   generatedDir,
+  // });
```
Comment (EdouardDem) [inline libs/schematics/src/generators/traxion-workspace/helpers/create-nestjs-application.helper.ts:19]: Is it intended that this code remains commented on?

File: libs/schematics/src/generators/traxion-workspace/helpers/normalize-options.helper.ts [hunk 4]
```diff
@@ -1,39 +1,59 @@
...
-    generatedImportPath: getImportPrefixPath(tree, generatedDir),
-    workspaceName: options.name,
-    parsedTags,
-    packageVersion,
-    uuid4: uuid4(),
+
+    // adminName: 'admin',
+    // apiName: 'api',
+    // pwaName: 'pwa',
+    // uuid4: uuid4(),
```
Comment (EdouardDem) [inline libs/schematics/src/generators/traxion-workspace/helpers/normalize-options.helper.ts]: Is it intended that this code remains commented on?

File: libs/schematics/src/generators/traxion-workspace/schema.d.ts [hunk 5]
```diff
@@ -1,21 +1,33 @@
...
-  pwaName: string;
-  uuid4: string;
-  workspaceName: string;
-  parsedTags: string[];
+  // generatedDir: string;
+  // adminName: string;
+  // apiName: string;
+  // pwaName: string;
+  // generatedImportPath: string;
+  // uuid4: string;
```
Comment (EdouardDem) [inline libs/schematics/src/generators/traxion-workspace/schema.d.ts]: Is it intended that this code remains commented on?

**Review by EdouardDem — APPROVED** [review]

File: apps/docs/docs/get-started/index.md [hunk 6]
```diff
@@ -4,285 +4,97 @@ title: Get started
...
 ```
 
-Use the following options. Replace `my-project` by the project name.
-
-```text
-✔ What name would you like to use? · admin
-✔ What is the path of the react admin library? · @my-project/generated-react-admin
-✔ What is the path of the rext client library? · @my-rpoject/generated-rext-client
-```
+This command will open the modeling GUI on localhost:4200, where you can update your modeling.
```
Comment (maxmousse) [inline apps/docs/docs/get-started/index.md]: I think hapify is served on port 4800

File: libs/schematics/src/generators/traxion-workspace/helpers/create-admin-application.helper.ts (hunk 1 above)
Comment (maxmousse) [inline libs/schematics/src/generators/traxion-workspace/helpers/create-admin-application.helper.ts:15]: There is some unused code here

File: libs/schematics/src/generators/traxion-workspace/helpers/create-angular-application.helper.ts [hunk 7]
```diff
@@ -7,11 +7,11 @@ export async function createAngularApplication(
...
   options: NormalizedOptions,
 ) {
-  const { pwaName, generatedDir, apiName, directory } = options;
-  await pwaAppGenerator(tree, {
-    name: pwaName,
-    directory,
-    generatedDir,
-    apiName,
-  });
+  // const { pwaName, generatedDir, apiName, directory } = options;
```
Comment (maxmousse) [inline libs/schematics/src/generators/traxion-workspace/helpers/create-angular-application.helper.ts:14]: There is some unused code here

File: libs/schematics/src/generators/traxion-workspace/helpers/create-nestjs-application.helper.ts [hunk 8]
```diff
@@ -7,10 +7,10 @@ export async function createNestjsApplication(
   tree: Tree,
   options: NormalizedOptions,
 ) {
-  const { directory, apiName, generatedDir } = options;
-  await apiAppGenerator(tree, {
-    name: apiName,
-    directory,
-    generatedDir,
-  });
+  // const { directory, apiName, generatedDir } = options;
```
Comment (maxmousse) [inline libs/schematics/src/generators/traxion-workspace/helpers/create-nestjs-application.helper.ts:14]: There is some unused code here

File: libs/schematics/src/generators/traxion-workspace/schema.d.ts [hunk 9]
```diff
@@ -1,21 +1,33 @@
...
+  createApps: boolean;
+  useConfigs: boolean;
+  useGitaction: boolean;
+  useAllLibraries: boolean;
+  skipInstall: boolean;
+  skipFormat: boolean;
 }
 
 interface NormalizedOptions extends TraxionWorkspaceGeneratorSchema {
+  createApps: boolean;
```
Comment (maxmousse) [inline libs/schematics/src/generators/traxion-workspace/schema.d.ts]: Some options are re-explicited but already present in `TraxionWorkspaceGeneratorSchema`

File: libs/schematics/src/generators/traxion-workspace/schema.d.ts [hunk 10]
```diff
@@ -1,21 +1,33 @@
...
   libsDir: string;
-  generatedDir: string;
-  generatedImportPath: string;
-  adminName: string;
-  apiName: string;
-  pwaName: string;
-  uuid4: string;
-  workspaceName: string;
-  parsedTags: string[];
+  // generatedDir: string;
```
Comment (maxmousse) [inline libs/schematics/src/generators/traxion-workspace/schema.d.ts]: There is some unused code here

File: libs/schematics/src/schematics.constants.ts [hunk 11]
```diff
@@ -17,11 +65,34 @@ export const DEFAULT_IMPORT_REPLACEMENTS = {
...
 };
+export const DEFAULT_IMPLICIT_DEPENDENCIES: Record<
+  AvailableTraxionTemplates,
+  string[]
+> = {
+  'angular-rext-client': ['common-rext-client'],
+  casl: [],
+  models: [],
+  'nestjs-models': ['nestjs-models-common', 'nestjs-models-rest'],
+  'nestjs-graphql': ['nestjs-models-common'],
```
Comment (maxmousse) [inline libs/schematics/src/schematics.constants.ts]: graphql depends on common-casl no ?

**Review by maxmousse — APPROVED** [review]

**Pull request merged**